src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -606,8 +606,10 @@ ReactDOMComponent.Mixin = {
     }
 
     switch (this._tag) {
-      case 'button':
       case 'input':
+        ReactDOMInput.mountReadyWrapper(this);
```

Comment:
Am I missing something here? It appears execution still continues after the markup is created to this call, even on the server-side.